js/core/component_registrator.d.ts

```diff
@@ -7,7 +9,7 @@
  * @namespace DevExpress
  * @hidden
  */
-export default function registerComponent(name: string, componentClass: any): void;
+export default function registerComponent(name: string, componentClass: Component): void;
```

Comment:
This seems to be a class not an instance:

```suggestion
export default function registerComponent(name: string, componentClass: typeof Component): void;
```